apps/docs-app/docs/experiments/sfc/index.md

```diff
@@ -0,0 +1,373 @@
+
+Together, it combines:
+
+- Colocated template, script, and style
+- No decorators
+- Performance-first defaults (`OnPush` change detection, no accesss to `ngDoCheck`, etc.)
+
+Here's a demonstration of the Analog format building a simple todo list:
+
+```html
+<script lang="ts">
+  import { signal } from '@angular/core';
+
+  let id = 0;
+
+  // Optional for most components
+  defineMetadata({
+    selector: 'app-root',
+  });
+
+  interface TodoItem {
+    id: number;
+    name: string;
+    done: boolean;
+  }
+
+  const list = signal<TodoItem[]>([]);
+
+  function onSubmit(e: SubmitEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+    const formData = new FormData(e.target);
+    const name = formData.get('name') as string;
+    list.update((prevList) => [
+      ...prevList,
+      {
+        name,
+        done: false,
+        id: ++id,
+      },
+    ]);
+  }
+
+  function onDone(id: number) {
+    list.update((prevList) =>
+      prevList.map((item) => {
+        if (item.id === id) {
+          return {
+            ...item,
+            done: !item.done,
+          };
+        }
+        return item;
+      })
+    );
+  }
+</script>
+
+<template>
+  <div class="container">
+    <h1>Todo list</h1>
+    <ul class="todoList">
+      @for (item of list(); track item.id) {
+      <li class="todoItem">
+        <span>{{ item.name }}</span>
+        <input
+          type="checkbox"
+          [checked]="item.done"
+          (change)="onDone(item.id)"
+        />
+      </li>
+      }
+    </ul>
+    <form (submit)="onSubmit($event)">
+      <h2>Add a task</h2>
+      <label>
+        <div>Task name</div>
+        <input name="name" />
+      </label>
+      <button>Add task</button>
+    </form>
+  </div>
+</template>
+
+<style>
+  .container {
+    max-width: 600px;
+    margin: 0 auto;
+    padding: 1rem;
+    text-align: center;
+  }
+
+  .todoList:empty::after {
+    content: 'No tasks';
+  }
+
+  .todoItem {
+    display: flex;
+    justify-content: space-between;
+  }
+</style>
+```
+
+# Usage
+
+To use the Analog SFC format, you'll need to use the Analog Vite plugin or the [Analog Astro plugin](/docs/packages/astro-angular/overview) with an additional flag to enable its usage:
+
+```typescript
+import { defineConfig } from 'vite';
+import analog from '@analogjs/vite-plugin-angular';
+
+export default defineConfig({
+  // ...
+  plugins: [
+    analog({
+      experimental: {
+        // Required to use the Analog SFC format
+        supportAnalogFormat: true,
+      },
+    }),
+  ],
+});
+```
+
+## IDE Support
+
+To support syntax highlighting and other IDE functionality with `.analog` files, you'll need to install an extension to support the format for:
+
+- [WebStorm 2024.1+ or IDEA Ultimate 2024.1+](https://github.com/analogjs/idea-plugin)
+
+> [Support for VSCode is coming! Please see this issue for more details](https://github.com/volarjs/angular-language-tools/).
```

Comment:
Do you think we should point to the [workaround](https://github.com/analogjs/analog/discussions/824#discussioncomment-8174883) for now?

Reply:
Personally? I'd vote no, as it creates a more "hacky" look and feel to the project IMO.

Reply:
I'm currently using this workaround, and I can't afford WebStorm yet, so it's nice to see a working option. I'd love to see dedicated support instead (I see you've already pinged about it in the other discussion), but while it's being worked upon, we could point to the existing alternative.

Would love to hear from others.